lib/cmd/fh3/notifications/list.js

```diff
@@ -0,0 +1,50 @@
+/* globals i18n */
+var common = require('../../../common.js');
+var fhreq = require("../../../utils/request");
+var util = require('util');
+
+module.exports = {
+  'desc' : i18n._('List notifications of a RHMAP App'),
+  'examples' : [{
+    cmd : 'fhc notifications list --app=<app> --env=<environment> --audit=<true>',
+    desc : i18n._('List notifications of the <app>')}],
+  'demand' : ['app', 'env'],
+  'alias' : {
+    'app':'a',
+    'env':'e',
+    'audit':'au',
+    'json': 'j',
+    0 : 'app',
+    1 : 'env',
+    2 : 'audit',
+    3 : 'json'
+  },
+  'describe' : {
+    'app'   : i18n._("Unique 24 character GUID of your cloud application."),
+    'env'   : i18n._("Unique 24 character GUID of the environment where this cloud application is deployed."),
+    'audit' : i18n._("Flag to list the notifications"),
```

Comment:
This parameter is passed to the REST as follows.

```
 if (params.audit) {
      payload.audit = true;
    }
```
I believe that it is a kind of filter in the REST and don't have any relation with the old API. I could not find different results with this parameter, so I couldn't understand the means to create a better description. However, I think we should keep this one to do the same what was done before.